scipy/linalg/basic.py

```diff
@@ -826,11 +830,14 @@ def lstsq(a, b, cond=None, overwrite_a=False, overwrite_b=False,
     if m != b1.shape[0]:
         raise ValueError('incompatible dimensions')
 
-    if lapack_driver not in ('gelsd', 'gelsy', 'gelss'):
-        raise ValueError('LAPACK driver "%s"is not found' % lapack_driver)
+    driver = lapack_driver
```

Comment:
`lapack_driver` could be changed to just `driver` above too.  This is all unreleased.